Makefile

```
@@ -0,0 +1,39 @@
+.PHONY: pull build down migrate-clickhouse migrate-clickhouse-reset integration generate-sql nuke-docker
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)